### DISCUSSION TITLE
List of fluids and phase equilibrium calculation

### DISCUSSION THREAD

**Discussion by fernandoczubinski** [pycalphad/pycalphad#577] [General]
Dear all, I was wondering if someone could help me. I would like to know where can I find a list of fluids which pycalphad contains and, in this case, if the pycalphad perform vapor-liquid-equilibrium (VLE) calculation (with PR EoS or any other cubic one). I would be interested in model a mixture composed by water, argon, hydrogen and silane. In addition, in the near future, probably perform the modeling with dichlorosilane, trichlorosilane e tetrachloridesilane (SiHxCly). Kind regards, Fernando

**Comment by bocklund** [comment]
Sorry this discussion went stale without an answer. PyCalphad doesn't provide any databases, so you would have to find or create one with suitable models for your problem.
